addons/crm_mail_client_extension/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'CRM Mail Client Extension',
+    'version': '1.0',
+    'category': 'Sales/CRM',
+    'sequence': 5,
+    'summary': 'Turn emails received in your mailbox into leads and log their content as internal notes.',
+    'description': "Turn emails received in your mailbox into leads and log their content as internal notes.",
+    'website': 'https://www.odoo.com/page/crm',
+    'depends': [
+        'web',
```

Comment:
Specifying web is not really necessary. I would give only explicit dependencies, crm, mail_client_extension, crp_iap_lead_enrich (in that case, don't forget to update migration scripts accordingly)